packages/mobile/src/screens/edit-collection-screen/EditCollectionNavigator.tsx

```diff
@@ -38,6 +43,10 @@ export const EditCollectionNavigator = (
       </Stack.Navigator>
       <PortalHost name='ConfirmPublishTrackPortal' />
```

Comment:
Can we remove this now?

Reply:
nice ty for the catch. i think from bad rebase